### PR TITLE
Adding exception for yaml includes in modules

### LINKED ISSUE #30: NoNestingInModules.yml shouldn't flag includes in codeblocks
image

### PR DESCRIPTION
**Pull request by rohennes**
Fixes #30

### CODE REVIEW

File: .vale/styles/OpenShiftAsciiDoc/NoNestingInModules.yml [hunk 1]
```diff
@@ -6,4 +6,4 @@ level: error
 link: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references
 message: "You can only nest snippets or Git Hub raw user content in modules."
 raw:
-  - 'include::(?!snippets|https:\/\/raw\.githubusercontent\.com).*(\[.*\])?'
+  - 'include::(?!snippets|https:\/\/raw\.githubusercontent\.com|.*\.yaml|.*\.yml).*(\[.*\])?'
```
Comment (aireilly) [inline .vale/styles/OpenShiftAsciiDoc/NoNestingInModules.yml:9]: .json, .txt too I think

**Review by aireilly — APPROVED** [review]
Small comment. Looks good!

File: .vale/styles/OpenShiftAsciiDoc/NoNestingInModules.yml (hunk 1 above)
Comment (aireilly) [reply .vale/styles/OpenShiftAsciiDoc/NoNestingInModules.yml:9]:
```
https:\/\/raw\.githubusercontent\.com\/.*\.(yaml|yml|json|txt|md|xml)
```

for good measure...

**Comment by rohennes** [comment]
Done @aireilly - good to merge you think?

File: .vale/styles/OpenShiftAsciiDoc/NoNestingInModules.yml [hunk 2]
```diff
@@ -6,4 +6,4 @@ level: error
 link: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references
 message: "You can only nest snippets or Git Hub raw user content in modules."
 raw:
-  - 'include::(?!snippets|https:\/\/raw\.githubusercontent\.com).*(\[.*\])?'
+  - 'include::(?!snippets|https:\/\/raw\.githubusercontent\.com|.*\.(yaml|yml|json|txt|md|xml)).*(\[.*\])?'
```
Comment (aireilly) [inline .vale/styles/OpenShiftAsciiDoc/NoNestingInModules.yml:9]:
```suggestion
  - 'include::(?!snippets).*\/*.*\.(adoc|asciidoc)\[.*\]'
```

Would this be more straightforward?  https://regexr.com/79ktq

**Comment by aireilly** [comment]
LGTM

**Pull request merged**